web-ui/src/main/resources/catalog/views/default/directives/partials/mdactionmenu.html

```diff
@@ -154,6 +154,17 @@
           <span data-translate="">enableWorkflow</span>
         </a>
       </li>
+      <!--metadata workflow assistance external link(s)-->
+      <li role="menuitem"
+          data-ng-if="isMdWorkflowEnable && md.isWorkflowEnabled()
+          && isMdWorkflowAssistEnable && user.canEditRecord(md)" data-ng-repeat="app in workFlowApps">
+        <a href=""
+           data-ng-href="{{app.appUrl.replace('{uuid}', md.getUuid()).replace('{isoLang}', iso2Lang)}}"
```

Comment:
Looke like you are missing `.replace('{lang}', lang)` for cases when users want to use 3char codes.

Reply:
I added the lang there. So it does support both 3 characters and 2 characters lang code.